test/PoseidonGoldenTests/GoldenTestsRunCommands.hs

```diff
@@ -1076,3 +1083,29 @@ testPipelineListRemote testDir checkFilePath = do
         ) (
         killThread threadID
         )
+
+testPipelineJannocoalesce :: FilePath -> FilePath -> IO ()
+testPipelineJannocoalesce testDir checkFilePath = do
+    let jannocoalesceOpts1 = JannoCoalesceOptions {
+            _jannocoalesceSource           = JannoSourceSingle "test/testDat/testJannoFiles/normal_full.janno",
+            _jannocoalesceTarget           = "test/testDat/testJannoFiles/normal_subset.janno",
+            _jannocoalesceOutSpec          = Just (testDir </> "jannocoalesce" </> "targetNoFieldsNoOverride.janno"),
+            _jannocoalesceFillColumns      = [],
+            _jannocoalesceOverwriteColumns = False
+        }
+    runAndChecksumFiles checkFilePath testDir (testLog $ runJannocoalesce jannocoalesceOpts1) "jannocoalesce" [
+          "jannocoalesce" </> "targetNoFieldsNoOverride.janno"
+        ]
+    -- runAndChecksumFiles checkFilePath testDir (testLog $ runJannocoalesce jannocoalesceOpts2) "jannocoalesce" [
+    --       "jannocoalesce" </> "targetFieldsNoOverride.janno",
+    -- ]
+    -- runAndChecksumFiles checkFilePath testDir (testLog $ runJannocoalesce jannocoalesceOpts3) "jannocoalesce" [
+    --       "jannocoalesce" </> "targetNoFieldsOverride.janno",
+    -- ]
+    -- runAndChecksumFiles checkFilePath testDir (testLog $ runJannocoalesce jannocoalesceOpts4) "jannocoalesce" [
+    --       "jannocoalesce" </> "targetFieldsOverride.janno",
+    -- ]
+    -- runAndChecksumFiles checkFilePath testDir (testLog $ runJannocoalesce jannocoalesceOpts5) "jannocoalesce" [
+    --       "jannocoalesce" </> "targetFromBaseDir.janno",
+    -- ]
```

Comment:
Is this work in progress? I can offer to fill them.

Reply:
I removed all those other gold tests for now, since I've added more unit tests to check various cases in `JannoccoalescenceSpec.hs`, which I think are easier to maintain. Of course, if you feel like adding more gold tests, feel free!